Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:10-alpine
+FROM node:11-alpine
```

Comment:
Latest version is `node:12.4-alpine`

https://hub.docker.com/_/node/

Reply:
Haha it was still 11.x.x when I submitted this PR
Fixed